tests/check_return_code.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+RET=0
+
+Check_return_code () {
+	EXPECTED="${1}"
+	shift 1
+
+	echo "Calling nsntrace ${*} ..."
+
+	sudo ../src/nsntrace "${@}"
+
+	RESULT="${?}"
+
+	if [ "${EXPECTED}" != "${RESULT}" ]
+	then
+		echo "nsntrace ${*} did not return with exit code ${EXPECTED} (saw: ${RESULT})"
+		RET="1"
+	fi
+}
+
+Check_return_code 0 /bin/true
+Check_return_code 1 /bin/false
+Check_return_code 1 /bin/does-not-exist
+
```

Comment:
Thanks!

Small nit: Could it be check_return_code, instead of Check_return_code?